web/src/components/Contact.tsx

```diff
@@ -28,17 +29,10 @@ interface Props {
 
 const Contact = (props: Props) => {
   const productName = props.productName
-
+  console.log(productName)
```

Comment:
Can this be removed?

Reply:
Removed in: https://github.com/bcgov/wps/pull/3557/commits/df7c95d01c40578f4652f107f2557febb581bd4c